corehq/apps/app_manager/views/utils.py

```diff
@@ -290,7 +288,6 @@ def handle_custom_icon_edits(request, form_or_module, lang):
             form_or_module.custom_icons = []
 
 
-@task(serializer='pickle', queue='background_queue', ignore_result=True)
```

Comment:
whats the reason for not moving the complete method instead?

Reply:
It was quicker and there wasn't a need to worry about moving around extra imports. Feel free to change this as wanted as long as the task is specified in the tasks file